### PR TITLE
rendercheck: version bumped to 1.5

### PR DESCRIPTION
**Pull request by esselfe**

### CODE REVIEW

File: xorg-app/rendercheck/DETAILS [hunk 1]
```diff
@@ -1,12 +1,12 @@
            SPELL=rendercheck
              PKG=rendercheck
```
Comment (vaygr) [inline xorg-app/rendercheck/DETAILS:2]: this is probably not needed

File: xorg-app/rendercheck/DETAILS [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-         VERSION=1.3
+         VERSION=1.5
         XORG_VER=X11R7.1
           SOURCE=${PKG}-${VERSION}.tar.bz2
```
Comment (vaygr) [inline xorg-app/rendercheck/DETAILS:5]: `PKG` can be replaced with `SPELL`

File: xorg-app/rendercheck/DETAILS [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-   SOURCE_URL[0]=ftp://ftp.x.org/pub/individual/app/$SOURCE
-     SOURCE_HASH=sha512:711083adbace701979bb56b708eab004f4e421ba0ea919d636abdfd3ad0ce98c9b0c3cd0c0320bcf4c2ab39f9ae254047c9cffc9b061a81bde262f1106a2b9e3
+   SOURCE_URL[0]=https://xorg.freedesktop.org/releases/individual/app/${SOURCE}
+     SOURCE_HASH=sha512:ffa1abf317f85a8bcef3544e3f810b34567553524ac0a784d2a67470e0811854d7983af8600a04aadecf42871ccb83ad5c0e9e25179142fda2a8f5853a07f616
 SOURCE_DIRECTORY=${BUILD_DIRECTORY}/${PKG}-${VERSION}
```
Comment (vaygr) [inline xorg-app/rendercheck/DETAILS:8]: same as above for `PKG`

File: xorg-app/rendercheck/DETAILS [hunk 4]
```diff
@@ -1,12 +1,12 @@
            SPELL=rendercheck
              PKG=rendercheck
-         VERSION=1.3
+         VERSION=1.5
         XORG_VER=X11R7.1
```
Comment (vaygr) [inline xorg-app/rendercheck/DETAILS:4]: I don't see this line being used anywhere

File: xorg-app/rendercheck/DETAILS [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
-   SOURCE_URL[0]=ftp://ftp.x.org/pub/individual/app/$SOURCE
-     SOURCE_HASH=sha512:711083adbace701979bb56b708eab004f4e421ba0ea919d636abdfd3ad0ce98c9b0c3cd0c0320bcf4c2ab39f9ae254047c9cffc9b061a81bde262f1106a2b9e3
+   SOURCE_URL[0]=https://xorg.freedesktop.org/releases/individual/app/${SOURCE}
+     SOURCE_HASH=sha512:ffa1abf317f85a8bcef3544e3f810b34567553524ac0a784d2a67470e0811854d7983af8600a04aadecf42871ccb83ad5c0e9e25179142fda2a8f5853a07f616
 SOURCE_DIRECTORY=${BUILD_DIRECTORY}/${PKG}-${VERSION}
-        WEB_SITE=http://xorg.freedesktop.org/wiki
+        WEB_SITE=https://xorg.freedesktop.org/wiki
          ENTERED=20060828
       LICENSE[0]=XCL
            SHORT="Module mkcomposecache for xorg"
```
Comment (vaygr) [inline xorg-app/rendercheck/DETAILS:12]: Both short and long descriptions should be updated.

**Comment by esselfe** [comment]
Closing with

**Pull request closed without merging**